### PR TITLE
Issue #9528: Update JavadocPackageCheckTest to use unique input files

### LINKED ISSUE #9528: update JavadocPackageCheckTest to use unique Input file in each test method
JavadocPackageCheckTest.java file reuse some Input files few time in few different test method.
Expected: each test method in Test class use it own Input file and do not share it with any other test method.

more details - https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests

### PR DESCRIPTION
**Pull request by bossever**
Fixes #9528

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage;
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage;      // violation
 
+/* Config:
+ * allowLegacy=false
```
Comment (nmancus1) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java:4]: Please use `"` as delimiter around property as you have done below for `fileExtensions="java"`. This goes for all following `allowLegacy` properties.

**Review by nmancus1 — CHANGES_REQUESTED** [review]
Minor item:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java (hunk 1 above)
Comment (bossever) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java]: Done.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java [hunk 2]
```diff
@@ -94,8 +94,8 @@ public void testBoth() throws Exception {
         };
         verify(createChecker(checkConfig),
             getPath("bothfiles" + File.separator + "InputJavadocPackageBothIgnored.java"),
-            getPath("bothfiles"
-            + File.separator + "InputJavadocPackageBothIgnored.java"), expected);
+            getPath("bothfiles" + File.separator + "InputJavadocPackageBothIgnored.java"),
+                expected);
```
Comment (nmancus1) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java:98]: Please move reformatting to different commit.

**Review by nmancus1 — CHANGES_REQUESTED** [review]
One more minor:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java (hunk 1 above)
Comment (nmancus1) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java]: Resolved.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java (hunk 2 above)
Comment (bossever) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java]: Done.

**Review by nmancus1 — APPROVED** [review]
Looks good. @strkkk please assign next reviewer.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java (hunk 2 above)
Comment (nmancus1) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java]: Resolved.

**Review by romani — CHANGES_REQUESTED** [review]
Update https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_input_suppressions.xml#L1170 to have no suppression

**Comment by bossever** [comment]
> Update https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_input_suppressions.xml#L1170 to have no suppression

Done.

**Review by romani — APPROVED** [review]
